Gemfile.jruby-1.9.lock

```diff
@@ -241,8 +241,8 @@ GEM
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       nokogiri
       xml-simple
-    logstash-input-beats (3.1.15-java)
-      concurrent-ruby (>= 0.9.2, <= 1.0.0)
+    logstash-input-beats (4.0.2-java)
```

Comment:
Good catch, I should have bumped to `3.1.18` which @ph released today to handle this gem incompatibility.